libraries/AP_MSP/AP_MSP.cpp

```diff
@@ -43,9 +43,9 @@ const AP_Param::GroupInfo AP_MSP::var_info[] = {
     // @Param: _OPTIONS
     // @DisplayName: MSP OSD Options
     // @Description: A bitmask to set some MSP specific options
-    // @Bitmask: 0:EnableTelemetryMode, 1: DJIWorkarounds
```

Comment:
discussed on call, better to remove the DJI option, and use backend choice

Reply:
@shellixyz any objections in removing the DJI Workaround option altogether and simply enable all DJI workarounds by default? Users should be unaffected

Reply:
No objection :+1: